source/_integrations/xiaomi_miio.markdown

```diff
@@ -1,37 +1,39 @@
 ---
-title: Xiaomi Miio
-description: Instructions how to integrate your Xiaomi Mi WiFi Repeater 2 within Home Assistant.
+title: "Xiaomi Miio"
+description: "Overvieuw of all Xiaomi Miio Integrations."
 ha_category:
-  - Presence Detection
+  - Hub
 ha_iot_class: Local Polling
-ha_release: 0.67
-ha_codeowners:
-  - '@rytilahti'
-  - '@syssi'
+ha_release: '0.110'
```

Comment:
This seems unlikely? This version number indicates the initial release of the integration, not the updated version number.

Please restore this to the original one.

Reply:
I now used the earliest version accorss all files that i consolidated.